x/staking/handler_test.go

```diff
@@ -956,7 +956,7 @@ func TestBondUnbondRedelegateSlashTwice(t *testing.T) {
 	got = handleMsgCreateValidator(ctx, msgCreateValidator, keeper)
 	require.True(t, got.IsOK(), "expected no error on runMsgCreateValidator")
 
-	// delegate 10 stake
+	// delegate 10 staking
```

Comment:
is the coin denom also changing ?

Reply:
no, the coin denom for tests should still be `stake`, this is an error, it should say `10 stake` here

Reply:
updated all errors of this form (3 in this file)